CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Progress bar should reflect latest design
 * [FEATURE] - [Share / Save consent form session](https://trello.com/c/zU3zqiy1/92-5-share-save-consent-form-session)
 * [BUG] - [Expenses / refreshments improvements](https://trello.com/c/2mT0SkHT/212-2-expenses-refreshments-improvements)
+* [CHORE] - Separate controllers into `ResearchSessionsController` and `QuestionsController`
```

Comment:
How come you have decided to add a changelog entry?

Reply:
Dangerbot decided for me. Path of least resistance.